docs/en/datasets/recog.md

```diff
@@ -209,13 +209,27 @@
 
 - Step1: Download [test_label.txt](https://download.openmmlab.com/mmocr/data/mixture/ct80/test_label.txt)
 
+- Step2: Download [timage.tar.gz](https://github.com/open-mmlab/mmocr/files/7408429/timage.tar.gz)
```

Comment:
@gaotongxiao Is this a permanent link?

Reply:
Or if you could provide a `download.openmmlab.com/` link I may update it.

Reply:
Sure, https://download.openmmlab.com/mmocr/data/mixture/ct80/timage.tar.gz

Reply:
> Sure, https://download.openmmlab.com/mmocr/data/mixture/ct80/timage.tar.gz

Thanks, updated in the closest commit.